packages/build-tools/src/buildErrors/buildErrorHandlers.ts

```diff
@@ -0,0 +1,100 @@
+import { BuildPhase, Platform } from '@expo/eas-build-job';
+import escapeRegExp from 'lodash/escapeRegExp';
+
```

Comment:
For now, it's mostly cache errors and a few I saw a lot in sentry. When this will go live I will have a better way to find new unknown errors, and make a second pass at adding unknown errors.